pkg/reconciler/isbsvc/installer/installer_test.go

```diff
@@ -193,6 +194,7 @@ func TestInstall(t *testing.T) {
 		testObj := testJetStreamIsbSvc.DeepCopy()
 		err := Install(ctx, testObj, cl, kubeClient, fakeConfig, zaptest.NewLogger(t).Sugar(), record.NewFakeRecorder(64))
 		assert.NoError(t, err)
+		testObj.Status.MarkChildrenResourceHealthy("RolloutFinished", "partitioned roll out complete: 3 new pods have been updated...")
```

Comment:
can we add one more unit test to cover the failure case where children resources are not healthy?

Reply:
Added